content/podcast/2024/ep29_Nextflow-2023-2024.md

```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
+date=2024-01-23
+type=podcast
+subtype=Community highlights
```

Comment:
I'm surprised you put this in Community highlights, it's more about technical features?

Reply:
Just copied it from our 2023 review (where I probably copied it from somewhere else). So far it looks like we've only ever done:

* Interview
* News & Views
* Community highlights (ep 27 only)

I'm not really sure if / where this tag is used, but I think that my original intention back when we rebooted Channels was to use this to differentiate between devrel team and Rob Lalonde.

```suggestion
subtype=subtype=News and Views
```

Reply:
Ah it shows on the listing page. I guess we can use whatever then. Feeling inspired for a new tag?

Reply:
```suggestion
subtype=Technical discussion
```

Reply:
```suggestion
subtype=Tech deep dive
```

Reply:
```suggestion
subtype=Nextflow nitty-gritty
```